app/packages/core/src/components/Sidebar/Entries/RegularEntry.tsx

```diff
@@ -63,7 +68,24 @@ const RegularEntry = React.forwardRef(
         title={title}
       >
         <Draggable color={color} entryKey={entryKey} trigger={trigger}>
-          <Header style={{ justifyContent: left ? "left" : "space-between" }}>
+          <Header
+            onMouseDown={(event: MouseEvent) => {
+              headerClickStart.current = event;
+            }}
+            onMouseUp={(event: MouseEvent) => {
+              if (!onHeaderClick) return;
+              const startX = headerClickStart.current.pageX;
```

Comment:
@brimoor re-added this bit of code to detect drag event.

Reply:
This is functional, of course, but it's an ambiguous click zone imo.